### ISSUE TITLE
Parsing schedule file with text-string (UDQ) as ALQ in WCONPROD fails

### ISSUE THREAD

**Issue by tklov11** [OPM/opm-common#2274]
Parsing schedule-files with UDQ for ALQ in WCONPROD fails with the following message:

_Problem with keyword WCONPROD in <memory string> line 1.
Internal error: Malformed floating point number 'UDA'"_

The schedule-file keyword content looks like this:
WCONPROD
A1 SHUT GRUP 2000.0 5000.0 10000.0 5000.0 1* 35.0 10.0 61 UDA 8* /
/

**Comment by joakim-hove** [comment]
Yes - that is right; UDQ values are currently only supported for the rate items in WCONxxxx. Absolutely some effort required to support UDQ for ALQ.

**Comment by joakim-hove** [comment]
This turned out be simpler than anticipated. Fix here: #2320
